public/app/plugins/datasource/cloudwatch/datasource.ts

```diff
@@ -224,6 +226,42 @@ export class CloudWatchDatasource extends DataSourceApi<CloudWatchQuery, CloudWa
     );
   }
 
+  private addDataLinksToResponse(response: DataQueryResponse, options: DataQueryRequest<CloudWatchQuery>) {
```

Comment:
nitpick: would change this to `addDataLinksToLogsResponse`. We are doing this also somewhere for metrics response so better not make this confusing.

Reply:
good catch, will change